Assets/HoloToolkit/Input/Scripts/Focus/RegisterPointableCanvas.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// Script to register a Canvas component so it's capable of being focused at for 'point and commit' scenarios.
+    /// </summary>
+    public class RegisterPointableCanvas :
```

Comment:
We really shouldn't need to register our canvas'.  Looking for a way to remove this.